.binder/environment.yml

```diff
@@ -46,6 +46,7 @@ dependencies:
   - ipympl
   - ipyvue
   - ipyvuetify
+  - plotly
```

Comment:
probably add a bottom pin, e.g.
```
- plotly >=5.12.0
```
... but this will force a conflict, because 

https://github.com/conda-forge/plotly-feedstock/pull/118